main.js

```diff
@@ -11,14 +11,16 @@ app.get('/api/test/cases', (req, res) => {
     res.send(testCasesRepository.getTestCases());
 });
 
+const isTestCaseValid = (title) => {
+    return !!title && !!title.trim();
+};
```

Comment:
W pliku `main.js` znajduje się już obsługa endpointów (nie jest to miejsce na to) oraz samo uruchomienie aplikacji. Dorzucenie jeszcze tego typu logiki tutaj mocno zaniży czytelność tego pliku (a jest głównym plikiem więc powinno być w miarę czysto i przejrzyście). 

Wydziel to do osobnego pliku, albo zaimplementuj tą logikę w `TestCasesRepository`.